tests/test-extensions/heisenberg-extension/src/main/java/org/mule/test/heisenberg/extension/HeisenbergOperations.java

```diff
@@ -611,10 +609,25 @@ public void close(HeisenbergConnection connection) throws MuleException {
     };
   }
 
-  public HeisenbergExtension getConfig(@Config HeisenbergExtension config, @Optional(defaultValue = "false") boolean fail) {
-    if (fail) {
-      throw new MuleRuntimeException(createStaticMessage("Failed returning the config"));
-    }
-    return config;
+  @MediaType(value = TEXT_PLAIN, strict = false)
```

Comment:
Why do we remove the old operation?

Reply:
I added to test https://www.mulesoft.org/jira/browse/MULE-18984 and get the config used. But this cannot be done for a streamed operation and also I found another way of getting the config during the tests `config = (HeisenbergExtension) muleContext.getExtensionManager().getConfiguration("heisenbergWithShortExpiration", event).getValue();` making this operation unnecessary. So I removed it to prevent the connector from getting even more bloated with operations and stuff.